Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -466,131 +465,107 @@ void USpatialActorChannel::SetChannelActor(AActor* InActor)
 	}
```

Comment:
Concept of `bCoreActor` should go away

Reply:
Guess singletons still shouldn't be deleted so we could just call it that.

Reply:
I think in this instance core actor refers to if we have a type binding for it or not.

Reply:
Shouldn't the concept of type bindings be killed completely as part of this PR (the only prerequisite is that we must generate the schema files for _all_ classes before starting a deployment).

Reply:
We still require you to tag your class with the `SpatialType` flag. We'll be moving to an opt-out path soon.